src/cli/README_UDP.md

```diff
@@ -95,6 +96,34 @@ Specifies the peer with which the socket is to be associated.
 Done
 ```
 
+### linksecurity
+
+Indicates whether the link security is enabled or disabled.
+
+```bash
+> linksecurity
```

Comment:
Looks like it should be `udp linksecurity`.

Reply:
Updated